pkg/util/cluster_info.go

```diff
@@ -20,6 +21,7 @@ func NewClusterInfo() *ClusterInfo {
 	info := &ClusterInfo{
 		esxiVersions: make(map[string]ESXiVersionInfo),
 		hwVersions:   make(map[string]int),
+		cbtEnabled:   make(map[bool]int),
```

Comment:
It is bit odd to see use of boolean as keys for a map. I understand that, this is for tracking number of nodes which has cbt enabled and number of nodes which has cbt disabled. Can we use strings here?

Reply:
All adjust accordingly to the above question.